belizairebr25.md

```diff
@@ -0,0 +1,50 @@
+
+**The display:**
+
+The plan was to use a transparent OLED as a monitor for a raspberry pi, but the technology for transparent oled's has not progressed far enough to function as a complete display for this project. I will use an LCD screen as a monitor for the raspberry pi, and swap it out for the transparent monitor when using it as a teleprompter, python console, text veiwer, etc...
+
+**The keyboard:**
+
+The keyboard is where the project gets more interesting. Ever since I dreamed of the future as a little child, I have envisioned wearable computers prolific as the iphone is today, but the keyboard and mouse have always eluded me. I finally solved the issue theoretically and can't wait to bring it to life. Essentially, where the joints are on the fingers there exists a line on the inside of the hand. This divides each finger into four areas, the tip, first knuckle, second knuckle, and the base on the palm. If a glove is worn with conductive peices on each space throughought both hands, there will be enough isolated areas for use as a keyboard(provided shift, number mode, and special charachter mode options are utilized). Then the thumb could have a contact wired to the positive of a low-voltage dc power supply. When the thumb contacts a part of the finger the circut is closed. Each section of the finger on each hand will have different resistance value attached to it and an arduino will read the resistance value of the circut knowing which value to assign to each charachter. Then three transistors per hand will be able to change which input is being used on the arduino and associates the same resistance with a number or capital for instance. The gate pin of these transistors will be wired to the modifying space of each hand allowing a different keyboard set for the opposite hand. These values will then be sent to the raspberry pi. As for the mouse, a regular wireless mouse can be used proided there is a flat surface present.
+
+**Why:** I give a lot of pre-written talks/speeches and it would be nice to have a teleprompter there to guide me if I forget where I am. It has also always been a dream of mine to make something like this and it would be awesome if that could become a reality.
+
+## Plan
+
+1. Design wiring schematics and paper drafts to guide when wiring and to find any potential issues with the idea, as well as to aid in knowing quantities of what is needed.
+2. Gather materials to construct the display and keyboard
+3. begin making keyboard prototypes with transistors, resistors, and arduino.
+4. Assemble the display and print the glasses frame.
```

Comment:
Will you be creating your own 3D model for the frame, or using an open source model? Could you clarify this in your project proposal?